keystores/src/main/java/com/palantir/conjure/java/config/ssl/KeyStores.java

```diff
@@ -317,6 +323,48 @@ static KeyStore newKeyStoreWithEntry(KeyStore original, Optional<String> passwor
         }
     }
 
+    /**
+     * Return a new {@link KeyStore} that contains the contents of the trust store and all default ca certificates.
+     *
+     * @param trustStorePath The path to the trust store.
+     * @param trustStoreType The type of trust store.
+     * @return a newly constructed key store of the type trustStoreType that contains the contents of the trust store
+     * and all default ca certificates.
+     */
+    public static KeyStore getCombinedTrustStoreAndDefaultCas(Path trustStorePath, StoreType trustStoreType) {
```

Comment:
Unsure if it would be preferred to put this in a different class rather than making this public.